docker/rootless/usr/bin/setup

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+if [ ! -f ${GITEA_CUSTOM}/conf/app.ini ]; then
+    mkdir -p ${GITEA_CUSTOM}/conf
+
+    # Set INSTALL_LOCK to true only if SECRET_KEY is not empty and
+    # INSTALL_LOCK is empty
+    if [ -n "$SECRET_KEY" ] && [ -z "$INSTALL_LOCK" ]; then
+        INSTALL_LOCK=true
+    fi
+
+    # Substitude the environment variables in the template
+    APP_NAME=${APP_NAME:-"Gitea: Git with a cup of tea"} \
+    RUN_MODE=${RUN_MODE:-"dev"} \
+    RUN_USER=${USER:-"git"} \
+    SSH_DOMAIN=${SSH_DOMAIN:-"localhost"} \
+    HTTP_PORT=${HTTP_PORT:-"3000"} \
```

Comment:
Why not common `8080` port?

Reply:
3000 is the default port already for gitea

Reply:
Why the default port not 8080?

Reply:
3000 is a common port used by golang projects for web servers, and when the codebase was started that was chosen as the port likely due to that. Gitea is primarily run behind a reverse proxy, so defaulting to a port that another webserver would be running on would cause conflicts. Other projects also follow similar choices, for example django (a python web framework) specifically avoids 80 and 8080 for this reason as it doesn't want a conflict, and is meant to be run behind a reverse proxy.

Reply:
One goal to this image is to allow us to break things. Even if I am pretty sure that most will vote 3000 we can put that to a vote.

Reply:
I created issue #10162 to discuss this subject.

Reply:
Even though this image does plan on eventually be breaking, changing the port just for this image would create a confusing situation where the default port for the binary is different from this image.